### PR TITLE
[ENG-125][ENG-804] Clean up unused EZID code/tidy up DOI creation and updating workflow

### PR DESCRIPTION
**Pull request by Johnetordoff**
## Purpose

We no longer use EZID, but we still have a lot of old EZID code, so we can get rid of it now.

## Changes

- removes EZID client
- removes old metadata generation code for crossref
- removes status from functions updating nodes/preprints

## QA Notes

Recommend checking creating DOI's for nodes and preprints, make sure nothing was accidentally broken.  (a lot of old code removed)

## Documentation

Not user facing, no docs

## Side Effects

None that I know of.

## Ticket
https://openscience.atlassian.net/browse/ENG-125

### CODE REVIEW

File: website/identifiers/metadata.py [hunk 1]
```diff
@@ -1,127 +0,0 @@
...
-        E.resourceType('Project', resourceTypeGeneral='Text'),
-        E.identifier(doi, identifierType='DOI'),
-        E.creators(*creators),
-        E.titles(E.title(remove_control_characters(title))),
-        E.publisher(publisher),
-        E.publicationYear(str(publication_year)),
-    )
-    # set xsi:schemaLocation
-    root.attrib['{%s}schemaLocation' % XSI] = SCHEMA_LOCATION
-    return lxml.etree.tostring(root, pretty_print=pretty_print)
```
Comment (pattisdr) [inline website/identifiers/metadata.py]: Can you tell me what we used to use this for, and what you're using now instead? Same with `datacite_metadata_for_node`?

File: website/identifiers/utils.py [hunk 2]
```diff
@@ -74,58 +47,36 @@ def request_identifiers(target_object):
...
         identifiers = client.get_identifier(doi)
-        already_exists = True
-        only_doi = False
     except exceptions.ClientResponseError as error:
         raise HTTPError(error.response.status_code)
     return {
-        'doi': identifiers.get('doi'),
-        'already_exists': already_exists,
-        'only_doi': only_doi
+        'doi': identifiers['doi']
```
[the remark this answers is not in the document]
Comment (Johnetordoff) [reply website/identifiers/utils.py]: Yes

File: website/identifiers/metadata.py (hunk 1 above)
Comment (Johnetordoff) [reply website/identifiers/metadata.py]: This was originally conceived as part of SHARE I believe. OSF and SHARE were going to share these xml parsing tools, but it never materialized.

File: website/identifiers/utils.py [hunk 3]
```diff
@@ -74,58 +47,36 @@ def request_identifiers(target_object):
...
+    except exceptions.IdentifierAlreadyExists:
         identifiers = client.get_identifier(doi)
-        already_exists = True
-        only_doi = False
     except exceptions.ClientResponseError as error:
         raise HTTPError(error.response.status_code)
     return {
-        'doi': identifiers.get('doi'),
-        'already_exists': already_exists,
-        'only_doi': only_doi
```
[the remark this answers is not in the document]
Comment (Johnetordoff) [reply website/identifiers/utils.py]: :+1:

File: website/identifiers/utils.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (pattisdr) [reply website/identifiers/utils.py]: i'd do identifiers.get('doi', None) to be safe

**Pull request merged**